OpenRA.Mods.Common/Lint/CheckTranslationReference.cs

```diff
@@ -70,40 +70,40 @@ void ILintMapPass.Run(Action<string> emitError, Action<string> emitWarning, ModD
 				return;
 
 			// TODO: Check all available languages.
-			var language = "en";
-			var modTranslation = new Translation(language, modData.Manifest.Translations, modData.DefaultFileSystem, _ => { });
-			var mapTranslation = new Translation(language, FieldLoader.GetValue<string[]>("value", map.TranslationDefinitions.Value), map, error => emitError(error.ToString()));
+			const string Language = "en";
```

Comment:
this won't should not be a constant, this value will need to come from other settings. It's a bit weird to change it

Reply:
As soon as it comes from other settings this line will need to go anyway, so we might as well do it like this.

Reply:
Yes indeed, changing it now so we can apply the rule doesn't preclude somebody changing it back in future.